### ISSUE TITLE
Tracking issue for all layers

### ISSUE THREAD

**Issue by fchollet** [keras-team/keras-core#53]
## Core layers -> @fchollet

- [x] Dense
- [x] Dropout
- [x] Activation
- [x] EinsumDense
- [x] Embedding
- [x] Identity
- [x] Masking

## Regularization layers -> @fchollet

- [x] ActivityRegularization
- [x] Dropout
- [x] GaussianDropout
- [x] GaussianNoise
- [x] SpatialDropout1D/2D/3D

## Attention -> @mattdangerw 

- [x] Attention
- [x] AdditiveAttention
- [x] MultiHeadAttention

## Activations

- [x] Elu
- [x] ReLU
- [x] LeakyReLU
- [x] PReLU
- [x] Softmax

## Merging -> @AakashKumarNain 

- [x] Add
- [x] Subtract
- [x] Average
- [x] Concatenate
- [x] Maximum
- [x] Minumum
- [x] Dot
- [x] Multiply

## Reshaping -> @hertschuh 

- [x] Flatten
- [x] Reshape
- [x] RepeatVector
- [x] Permute
- [x] Cropping1D/2D/3D
- [x] ZeroPadding1D/2D/3D
- [x] UpSampling1D/2D/3D

## Convolutional -> @chenmoneygithub 

- [x] Conv1D/2D/3D
- [x] SeparableConv1D/2D
- [x] DepthwiseConv1D/2D
- [x] Conv1DTranspose/2D/3D

## Pooling -> @chenmoneygithub 

- [x] AveragePooling1D/2D/3D
- [x] MaxPooling1D/2D/3D
- [x] GlobalAveragePooling1D/2D/3D
- [x] GlobalMaxPooling1D/2D/3D

## Normalization

- [x] BatchNormalization
- [x] UnitNormalization
- [x] LayerNormalization
- [x] GroupNormalization
- [x] SpectralNormalization

## RNN -> @fchollet + @LukeWood 

- [x] RNN / RNNCell
- [x] SimpleRNN
- [x] LSTM
- [x] GRU
- [x] ConvLSTM1D/2D/3D
- [x] TimeDistributed
- [x] Bidirectional

## Preprocessing layers -> @fchollet

TBD
